charts/console/Chart.yaml

```diff
@@ -44,4 +44,4 @@ annotations:
       url: https://helm.sh/docs/intro/install/
   artifacthub.io/images: |
     - name: redpanda
-      image: vectorized/console:v2.0.2
+      image: vectorized/console:v2.1.1
```

Comment:
```suggestion
      image: docker.redpanda.com/vectorized/console:v2.1.1
```

Reply:
Not according to https://github.com/redpanda-data/helm-charts/blob/main/charts/console/values.yaml#L9